static_src/test/functional/user_role.spec.js

```diff
@@ -1,58 +1,150 @@
         cookieResult = browser.getCookie('testing_user_role').value;
-        expect(cookieResult).toBe('space_manager_space_xx');
+        expect(cookieResult).toBe(cookieManagerSpaceXX);
       });
     });
   });
 
-  describe('Org page', function () {
-    beforeEach(function () {
-      browser.url('/#/org/user_role-org_x-ffe7-4aa8-8e85-94768d6bd250');
+  describe('Org page as org manager X', function () {
+    describe('if org manager Y then they', function () {
+      it('should navigates to org Y ', function () {
+        browser.url(urlOrgY);
+
+        browser.waitForExist('.test-users');
+        userRoleElement = new UserRoleElement(browser, browser.element('.test-users') );
+
+        expect(userRoleElement.isVisible()).toBe(true);
+      });
+
+      it('should not be able to edit roles for org Y', function () {
+        expect(userRoleElement.isUserOrgManager(guid_manager_orgX)).toBe(false);
+        expect(userRoleElement.isUserOrgManager(guidManagerOrgY)).toBe(true);
+      });
     });
 
-    it('has a title', function () {
-      expect(browser.getTitle()).toBe('cloud.gov dashboard');
+    describe('if manager for org X then they', function () {
+      it('should navigates to org X ', function () {
+        browser.url(urlOrgX);
+
+        browser.waitForExist('.test-users');
+        userRoleElement = new UserRoleElement(browser, browser.element('.test-users') );
+
+        expect(userRoleElement.isVisible()).toBe(true);
+      });
+
+      it('should be able to edit roles for org X', function () {
+        expect(userRoleElement.isUserOrgManager(guid_manager_orgX)).toBe(true);
+        expect(userRoleElement.isUserOrgManager(guidManagerOrgY)).toBe(false);
+      });
     });
+  });
 
-    it('has a page header', function () {
-      const pageHeader = browser.element('.test-page-header-title');
-      expect(pageHeader.getText()).toBe('fake-cf-user_role-org_x-testing');
+  describe('Testing user roles', function () {
+    it('Setup userRoleElement', function () {
+      browser.url(urlOrgX);
+      browser.waitForExist('.test-users');
+      userRoleElement = new UserRoleElement(browser, browser.element('.test-users') );
     });
 
-    describe('org manager for org X then they should', function () {
-      it('be able to edit roles for org X', function () {
-        browser.setCookie({ name: 'testing_user_role', value: 'org_manager_org_x' });
-        cookieResult = browser.getCookie('testing_user_role').value;
-        expect(cookieResult).toBe('org_manager_org_x');
+    describe('As org manager Y', function () {
+      it('set cookie to org Y manager', function () {
+        cookieValue = cookieManagerOrgY;
+      });
 
-        expect(true).toBe(true);
+      describe('shouldn\'t have permission to edit fields on org X pages', function () {
+        it('set url', function () {
+          pageUrl = urlOrgX;
+        });
+
+        it('verify cookie', function () {
```

Comment:
more description. something like: verifies that we are currently org manager X in this session